src/frontends/onnx/frontend/src/op/mod.cpp

```diff
@@ -4,32 +4,31 @@
 
 #include "op/mod.hpp"
 
-#include <memory>
-
-#include "default_opset.hpp"
 #include "exceptions.hpp"
-#include "ngraph/op/util/attr_types.hpp"
 #include "openvino/frontend/exception.hpp"
-#include "openvino/op/abs.hpp"
+#include "openvino/op/floor_mod.hpp"
+#include "openvino/op/mod.hpp"
+
+using namespace ov::op;
 
 OPENVINO_SUPPRESS_DEPRECATED_START
```

Comment:
do we need to keep `OPENVINO_SUPPRESS_DEPRECATED_START` ? Looks like we don't have deprecated API usage in refactored files

Reply:
I'll remove it in a separate PR when everything will be fixed